src/magicbox/SuggestionsManager.ts

```diff
@@ -94,6 +94,9 @@ export class SuggestionsManager {
     const selected = this.keyboardFocusedSuggestion;
     if (selected != null) {
       $$(selected).trigger('keyboardSelect');
+      // By definition, once an element has been "selected" with the keyboard,
+      // it is not longer "active" since the event has been processed.
+      this.keyboardFocusedSuggestion = null;
```

Comment:
I think it makes more sense to move this statement to the handler of `keyboardSelect` on line 220.